app/models/katello/docker_manifest.rb

```diff
@@ -2,7 +2,7 @@ module Katello
   class DockerManifest < Katello::Model
     include Concerns::PulpDatabaseUnit
 
-    has_one :docker_tag, :dependent => :destroy, :class_name => "Katello::DockerTag"
+    has_many :docker_tags, :dependent => :destroy, :class_name => "Katello::DockerTag"
```

Comment:
might need to update https://github.com/Katello/katello/blob/master/app/views/katello/api/v2/docker_manifests/show.json.rabl#L5